monitors/filterWords.js

```diff
@@ -0,0 +1,34 @@
+exports.run = async (client, msg) => {
+  if (!msg.guild) return;
+  const { modLog, swearWords } = msg.guild.settings;
+  if (!msg.deletable && !swearWords.length > 0) return;
+  const badWords = swearWords.filter(word => msg.content.toLowerCase().includes(word));
+  if (badWords.length > 0) {
```

Comment:
Another indentation that could be removed.

Reply:
Fixed but question, if indentations bad for any reason? performance possibly?

Reply:
Better code readability

Reply:
Thank you will keep it in mind for future